contracts/test/Verifiers.ts

```diff
@@ -50,14 +50,13 @@ describe("Verifier Contracts", () => {
       expect(await snarkVerifier.verifyProof(verifyingKey.address, commitmentCalldata.range_check_snark_proof, [1])).to.be.true;
     });
 
-    it("should fail to verify snark proof without the number of instances", async () => {
-      await expect(snarkVerifier.verifyProof(verifyingKey.address, commitmentCalldata.range_check_snark_proof, [])).to.be.reverted;
+    it("should revert grand sum proof", async () => {
+      await expect(snarkVerifier.verifyProof(verifyingKey.address, commitmentCalldata.grand_sums_batch_proof, [1])).to.be.reverted;
```

Comment:
Why should it revert here?

Reply:
Because the verifier is for verifying snark proof, not for verifying grand sum proof.

Reply:
I would rename the test to something like "should revert with invalid proof".